composer.json

```diff
@@ -0,0 +1,88 @@
+{
+    "name": "pheature-flags/pheature-flags",
```

Comment:
In this main `composer.json` we are using `pheature-flags` as package namespace, while in `composer.json` files within the `packages/` folder, `pheature` namespace is used. Is it because of something about monorepo transformation scripts? Shall we consider to make it consistent across all codebase? (either `pheature` or `pheature-flags`)